openpype/settings/entities/schemas/projects_schema/schema_project_deadline.json

```diff
@@ -117,6 +117,18 @@
                             "key": "asset_dependencies",
                             "label": "Use Asset dependencies"
                         },
+                        {
+                            "type": "number",
+                            "key": "priority",
+                            "label": "Priority",
+                            "default": 50
```

Comment:
Does having default in the schema make any sense if at all times the defaults are loaded anyway from the default settings? I don't recall ever seeing `default` set on objects in the schema.

Reply:
Probably doesn't make sense, fixed.